External/FEXCore/Source/Interface/IR/Passes/DeadContextStoreElimination.cpp

```diff
@@ -608,7 +619,14 @@ bool RCLSE::Run(FEXCore::IR::IREmitter *IREmit) {
   // XXX: We don't do cross-block optimizations yet
   //CalculateControlFlowInfo(IREmit);
   bool Changed = false;
-  Changed |= RedundantStoreLoadElimination(IREmit);
+  
+  VerboseLog("Start!");
+  while(RedundantStoreLoadElimination(IREmit)) {
```

Comment:
Can we bound this loop to some kind of max iterations. 

During some dev on top of this branch I ran into a case where this looped indefinitely.

Reply:
It'll run a maximum of 5 times now